### PR TITLE
RDISCROWD-7088 Ownership ID v2

### PR DESCRIPTION
**Pull request by n00rsy**
*Issue number of the reported bug or feature request: [RDISCROWD-7088](https://jira.prod.bloomberg.com/browse/RDISCROWD-7088)*

**Describe your changes**
 - v2 of https://github.com/bloomberg/pybossa-default-theme/pull/469
 - switch to `/api/project/<short_name>`

### CODE REVIEW

**Review by ciaracian — APPROVED** [review]
LGTM, thanks Noor!

File: templates/projects/summary.webpack.ejs [hunk 1]
```diff
@@ -13,6 +13,9 @@
     </ul>
     <br>
 
+    <div id = "ownership-id-title" style="display: none;">{{config.OWNERSHIP_ID_TITLE}}</div>
```
Comment (kbecker42) [inline templates/projects/summary.webpack.ejs:16]: Another option to consider is to use data attributes:

```html
<div id="ownership-attributes" data-title="{{config.OWNERSHIP_ID_TITLE}}" data-project-id="{{project.id}}" style="display: none;"></div>
```

**Comment by kbecker42** [comment]
We should coincide this PR with removal of the server-side API from https://github.com/bloomberg/pybossa/pull/978
Additionally, it would be useful to add a server-side validation check on the ownership ID to double for the client-side check.

**Comment by n00rsy** [comment]
> We should coincide this PR with removal of the server-side API from [bloomberg/pybossa#978](https://github.com/bloomberg/pybossa/pull/978) Additionally, it would be useful to add a server-side validation check on the ownership ID to double for the client-side check.

Yes, we've added server side validation with https://github.com/bloomberg/pybossa/pull/978. This PR will be merged in with https://github.com/bloomberg/pybossa/pull/978 before it's merged to main.

**Pull request merged**